desc/grid.py

```diff
@@ -664,14 +856,22 @@ def __init__(
         rho=np.array(1.0),
         theta=np.array(0.0),
         zeta=np.array(0.0),
+        coordinates="rtz",
```

Comment:
do we want this to be an option? I feel like for all our regular grids it might be better to hard-code the coordinates and period?

Reply:
I agree i'll change